stable/storage-class/templates/storageclass.yaml

```diff
@@ -73,11 +73,11 @@ parameters:
 {{- include "storageClass.allowVolumeExpansion" . }}
 reclaimPolicy: Retain
 volumeBindingMode: Immediate
-{{- else }}
+{{- end }}
+---
```

Comment:
this was an unrelated bug, correct?

Reply:
I wouldn't say it was a bug, I just think that local storage shouldn't be excluded when running on cloud (e.g local ssd)